examples/controls/README.md

```diff
@@ -0,0 +1,12 @@
+<div align="center">
+  <img src="https://avatars3.githubusercontent.com/u/2105791?v=3&s=200" />
```

Comment:
Reminds me that we still haven't made up our minds on where we want images to be stored... if in repo it is easier to maintain, but they do take some space.